it/config.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
This file is no longer needed with GBI-1123

Reply:
Was added to gitignore